edifact/models/tests/test_message.py

```diff
@@ -1,118 +1,161 @@
-import unittest

```

Comment:
None if these edifact tests run as part of the build because of missing __init__.py files

Reply:
init added